src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -193,15 +189,6 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
     return this._trigger || 'click';
   }
 
-  /**
-   * @internal
-   * Indicates if the dropdown button element or any of its children have focus.
-   * @deprecated This property will be removed in the next major version release.
-   */
-  public get buttonIsFocused(): boolean {
```

Comment:
I do think I'll be able to get focus to work without these, but I think the datepicker component might actually call these (I remember adding them here so I could call them from datepicker so we had a first class way of determining if things were focused). @Blackbaud-AlexKingman you're working on converting datepicker to the new popovers, right? I think you would need to remove the "isFocused" properties there too

Reply:
Correct. I will not need to access these properties anymore once my changes are released.